### DISCUSSION TITLE
Revisit examples folder structure

### DISCUSSION THREAD

**Discussion by YuanTingHsieh** [NVIDIA/NVFlare#936] [Ideas]
Right now we have hello-XXX examples which are applications that can be upload and deploy in NVFlare.

And other examples like "cifar", "prostate" are more like a walk through tutorial instead of an application.

I think we should separate these two concepts.

**Comment by YuanTingHsieh** [comment]
One proposal is that, 

We have a place to put all the example "apps", that is it only contains custom and config folder for each of them.

We then have another place called "tutorials", that can have all these readme and additional scripts to run the examples.

We can also have a place to put common scripts to run examples, instead of having duplicate scripts like "run_poc.sh" in every folder.

**Comment by holgerroth** [comment]
The example root [readme](https://github.com/NVIDIA/NVFlare/blob/main/examples/README.md) already tries to give some structure to the examples.

**Comment by YuanTingHsieh** [comment]
Thanks, I see that the readme file provide explanations to the examples folder.

But what I am thinking here is a more unified structure.

We could move the scripts (for example: run_poc.sh) in upper layer and then make it capable of running any example app such as hello-pt or cifar10_fedavg or prostate_ditto.

**Comment by chesterxgchen** [comment]
@chesterxgchen  need to discuss with other teams
